spec/pluralrules.html

```diff
@@ -347,7 +385,38 @@ <h1>ResolvePlural ( _pluralRules_, _n_ )</h1>
         1. Let _res_ be ! FormatNumericToString(_pluralRules_, _n_).
         1. Let _s_ be _res_.[[FormattedString]].
         1. Let _operands_ be ! GetOperands(_s_).
-        1. Return ! PluralRuleSelect(_locale_, _type_, _n_, _operands_).
+        1. Let _p_ be ! PluralRuleSelect(_locale_, _type_, _n_, _operands_).
+        1. Return the Record { [[PluralCategory]]: _p_, [[FormattedString]]: _s_ }.
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-pluralruleselectrange" aoid="PluralRuleSelectRange">
+      <h1>PluralRuleSelectRange ( _locale_, _type_, _xp_, _yp_ )</h1>
+
+      <p>
+        When the PluralRuleSelectRange abstract operation is called with four arguments, it performs an implementation-dependent algorithm to map the plural form String values _xp_ and _yp_, representing the plural forms of the start and end of a range, to a resolved String value for the plural form of the range as a whole denoted by _type_ for the corresponding _locale_, or the String value *"other"*.
+      </p>
+    </emu-clause>
+
+    <emu-clause id="sec-resolvepluralrange" aoid="ResolvePluralRange">
+      <h1>ResolvePluralRange ( _pluralRules_, _x_, _y_ )</h1>
+      <p>
+        When the ResolvePluralRange abstract operation is called with arguments _pluralRules_ (which must be an object initialized as a PluralRules), _x_ (which must be a Number value), and _y_ (which must be a Number value), it returns a String value representing the plural form of the range starting from _x_ and ending at _y_ according to the effective locale and the options of _pluralRules_. The following steps are taken:
```

Comment:
Why does ResolvePluralRange return a plural category String value rather than e.g. a { [[PluralCategory]], [[FormattedStartString]], [[FormattedEndString]] } Record to better align with ResolvePlural?

Reply:
I think because I don't need the extra info from `ResolvePluralRange`; I changed `ResolvePlural` to return a Record only because I needed the formatted string to perform some other operation.